src/zarr/codecs/__init__.py

```diff
@@ -21,5 +28,19 @@
     "ShardingCodec",
     "ShardingCodecIndexLocation",
     "TransposeCodec",
+    "VLenUTF8Codec",
+    "VLenBytesCodec",
     "ZstdCodec",
 ]
+
+
+def get_default_array_bytes_codec(
```

Comment:
Let's mark this as private for now

```suggestion
def _get_default_array_bytes_codec(
```